platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/ConfigMappings.java

```diff
@@ -35,7 +35,11 @@ private ConfigMappings() {}
     private static final List<ConfigMapping> MAPPINGS = List.of(
             new ConfigMapping("consensus.roundsNonAncient", "state.roundsNonAncient"),
             new ConfigMapping("consensus.roundsExpired", "state.roundsExpired"),
-            new ConfigMapping("consensus.coinFreq", "coinFreq"));
+            new ConfigMapping("state.coinFreq", "coinFreq"),
```

Comment:
this looks like a mistake, `state.coinFreq` does not seem to exist

Reply:
thanks for the hint :)